### ISSUE TITLE
References for RGD annotations missing

### ISSUE THREAD

**Issue by rarebres** [geneontology/go-annotation#2014]
Many gene:biological_process annotations submitted by RGD that were previously associated with a PMID reference in the past (ie. before mid-April) are currently cited in AmiGO without a PMID reference value.  Now, only RGD:1624291 is cited as reference.  The change affected about 50k annotations in a subset I recently analyzed.  The data available in the current RGD GAF files (eg. rattus_genes_go) have the previously associated PMID values.  Are those PMIDs being blocked or no longer being passed/parsed?

Eg. Zfp36 to GO:0000122 from PMID:21784977
Eg. Zfp36 to GO:0045085 from PMID:15634918 
Eg. Rab31 to GO:0045055 from PMID:17189207

**Comment by pgaudet** [comment]
@kltm @dougli1sqrd Some RGD references are indeed missing from AmiGO - can you please have a look ? 

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/4782928/45998012-bb555700-c0a1-11e8-99b1-e30b6e61845c.png">

**Comment by kltm** [comment]
Looking at what we have for the upstream RGD `source`:
```bash
wget http://geneontology.org/gene-associations/submission/gene_association.rgd.gz
zgrep PMID:21784977 gene_association.rgd.gz | wc -l
0
```
PMID:21784977 does not appear.

**Comment by pgaudet** [comment]
@rarebres We cannot find these PubMedIDs in your original GAF (http://snapshot.geneontology.org/annotations/index.html); neither is it is the source you provide here: http://geneontology.org/gene-associations/submission/

(Although I do see them in RGD). Can you please check that the annotations are correctly exported at your end ? 

Thanks, Pascale

**Comment by rarebres** [comment]
The files have been regenerated since my original observation of the presence of the PMIDs, and since my reporting of their absence.  After further investigation I found the explanation here: 
http://geneontology.org/gene-associations/readme/rgd.README
"=====March 2018=====
At the request of the GOC, and to comply with the GOC specifications for GAFs, information about the original references for ISO annotations were removed from the gene_association.rgd file's reference field (column 6).  "Original references" refers to (e.g.) PubMed or other reference IDs of publications used to make experimental annotations for orthologous human or mouse genes, from which ISO annotations were inferred to the corresponding rat genes.  Original references for ISO annotations can still be found on the RGD website (https://rgd.mcw.edu) and in files on the RGD ftp site (ftp://ftp.rgd.mcw.edu/pub/data_release/gene_protein_association.rgd.gz, as well as the ...go files in the ftp://ftp.rgd.mcw.edu/pub/data_release/annotated_rgd_objects_by_ontology/ directory and the ...bp, ...mf and ...cc files in the 'with terms' subdirectory).
"
That coincides perfectly with the apparent loss between February and May.  It suggests that one shouldn't observe PMIDs in combination with ISO evidence codes.  While those no longer appear for RGD submissions, I do see several thousand from other sources/species like MGI/mouse and a few others.
